$NetBSD: patch-configure.ac,v 1.1 2018/06/04 05:09:33 dbj Exp $

There is a typo in the gexiv2-0.10.8 configure.ac and configure

--- configure.ac.orig	2018-02-05 21:56:47.000000000 +0000
+++ configure.ac
@@ -220,7 +220,7 @@ case "$target_or_host" in
     ;;
 esac
 AC_MSG_RESULT([$platform_darwin])
-AM_CONDITIONAL(PLATFORM_DARWIN, test "$platform_win32" = "yes")
+AM_CONDITIONAL(PLATFORM_DARWIN, test "$platform_darwin" = "yes")
 
 
 #################
